pkg/http/http.go

```diff
@@ -361,7 +361,7 @@ func (c *Client) handleResponse(response *http.Response) (*http.Response, error)
 	case http.StatusForbidden:
 		c.logger.WithField("HTTP Error", "403 (Forbidden)").Error("Permission issue, please check your user permissions!")
 	case http.StatusNotFound:
-		c.logger.WithField("HTTP Error", "404 (Not Found)").Error("Requested resource could not be found")
+		c.logger.WithField("HTTP Error", "404 (Not Found)").Errorf("Requested resource ('%s') could not be found", url)
```

Comment:
@marcusholl can we use `response.Request.URL` instead of an additional parameter?

Reply:
I was not aware of response.Request. In case that contains the same we can also use that ... The important thing here is: we should know which URL we tried to access ...